### ISSUE TITLE
Drag and drop not working in Chrome

### ISSUE THREAD

**Issue by amiteshs** [ducksboard/gridster.js#236]
The drag and drop of elements are not working in Chrome version 30.0.1599.69 m (which is the latest version)

**Comment by coopr** [comment]
FWIW, drag and drop works in Chrome version 30.0.1599.69 on Mac OSX (a relatively recent version) and also seems to work in 30.0.1599.101 (latest version).

**Comment by kmelkon** [comment]
it's working for me in chrome  version 31.0.1650.26 beta on Ubuntu 13.04.

**Comment by amiteshs** [comment]
Hi guys
I tired with Chrome version 30.0.1599.69 m on Windows 8 where it does not work

**Comment by doczoidberg** [comment]
I've got the same problem on windows 8.1 chrome Version 31.0.1650.34 beta-m

**Comment by rangerdopp** [comment]
Just checked and it doesn't work on Chrome Version 30.0.1599.101 either.

**Comment by philoyang** [comment]
see here https://github.com/ducksboard/gridster.js/issues/141

I fixed it by adding styles [data-row=&quot;46&quot;] { top:13864px; } [data-row=&quot;45&quot;] { top:13556px; }....... into head

**Comment by msilb** [comment]
Same here, does not work on Chrome 34.0.1847.131, which is quite disturbing because this is the latest browser version there is. Works fine in IE 11 though.

**Comment by xfufux** [comment]
I can drag items using my touch screen but not the trackpad/mouse (tested on an Asus Zenbook w/ Win 8.1).  IE11 gets really messed up when trying to use the touch screen but the mouse works fine.

**Comment by msilb** [comment]
Yes, I found out why it was not working, looks like touch screen support automatically deactivates mouse dragging capabilities on Chrome (does not happen on IE11 for me).

**Comment by ajmajma** [comment]
@msilb I am having the same problem with my zenbook, is there a workaround for this?

**Comment by msilb** [comment]
@ajmajma not that i know of, you'd have to use your touch screen to drag tiles around on chrome, unfortunately. :(

**Comment by ghost** [comment]
I've encountered this same issue with Chrome and touchscreen supporting laptops.  For what it's worth, this other project has something figured out.

https://github.com/dbushell/Nestable/pull/66

**Comment by kutluarasli** [comment]
Drag/drop was not working in Chrome browser(36.0.1985.125). I have disabled the installed mouse software and everything works fine now. Is official solution to disable it at all? 

Edit: I had downloaded Gridster via nuget (Visual Studio package manager) which refers version "v0.5.1 - 2014-03-05". Upgraded to v0.5.2, now it work fine at all. Thanks vieron.

**Comment by msilb** [comment]
I can see it's fixed now, works fine in Chrome 36.0.1985.125 m. :thumbsup:

**Comment by vazy** [comment]
I'm still experiencing this in Chrome 40.0.2214.115 m :(
